contracts/proxy/Clones.sol

```diff
@@ -16,27 +18,34 @@ pragma solidity ^0.8.20;
  */
 library Clones {
     /**
-     * @dev A clone instance deployment failed.
+     * @dev Deploys and returns the address of a clone that mimics the behaviour of `implementation`.
+     *
+     * This function uses the create opcode, which should never revert.
      */
-    error ERC1167FailedCreateClone();
+    function clone(address implementation) internal returns (address instance) {
+        return clone(implementation, 0);
+    }
 
     /**
      * @dev Deploys and returns the address of a clone that mimics the behaviour of `implementation`.
      *
      * This function uses the create opcode, which should never revert.
      */
```

Comment:
Let's keep it simple:

```solidity
/**
* @dev Same as {clone}, but with a `value` parameter to send native currency to the new contract.
*/
```

Reply:
I'm also realizing every `{clone}` and `{cloneDeterministic}` reference will break in the docs now that there are two signatures. We need to confirm.

Reply:
Fixed here. There are no other references.